src/Stryker.Core/Stryker.Core/Reporters/DashboardReporter.cs

```diff
@@ -32,27 +34,40 @@ public class DashboardReporter : IReporter
 
         public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent, TestProjectsInfo testProjectsInfo)
         {
+            if (_options.ReportTypeToOpen == ReportType.Dashboard)
+            {
+                _dashboardClient.PublishFinished().Wait();
```

Comment:
What advantage does this have over the current implementation?

Reply:
Iirc we discussed this in our last meeting.

Right now real time reporting with is coupled to auto-open report. For the html reporter this makes sense, for the dashboard it does not.

Because you might want to use real-time reporting from your pipeline so you can follow the results during the run, but you obviously won't use auto-open report in the pipeline. 

My proposal is this:

- Add a new reporter option `RealTimeDashboard` (enum)
- In the DashboardReporter check if the `RealTimeDashboard` reporter is enabled, and/or that auto-open-report has been enabled with auto open report type Dashboard. In which case use real-time reporting.
- When real-time reporting is enabled - either through `RealTimeDashboard` or auto-open-report, print the report uri to the console from the start
- When auto-open-report is enabled, obviously auto open the report on the client machine

Effectively I think this is nothing more than adding the enum and extending the `if (_options.ReportTypeToOpen == ReportType.Dashboard)` condition to 

`if (_option.Reporters.Contains(Reporters.RealTimeDashboard) || _options.ReportTypeToOpen == ReportType.Dashboard)`

Reply:
Ah check, yep we discussed this. Seems like a small change indeed, will implement this.

Reply:
Should work now, but I will have to test this (for real) on my other laptop (will do that tomorrow)

Reply:
Works now, but had to make some changes to the input (when I added the new enum).